client/test/cypress/integration/register.spec.js

```diff
@@ -77,6 +63,28 @@ describe("Register", () => {
                 .parents('label')
                 .should('not.have.class', 'q-field--error')
 
+             //Username must be unique
+            cy.dataCy('username_field')
+                .clear()
+                .type('regularUser{enter}')
+                .parents('label')
+                .should('have.class', 'q-field--error')
+                 .contains('is not available');
+            cy.dataCy('username_field')
+                .type('brandnewusername')
+
+             //Email must be unique
+            cy.dataCy('email_field')
+             .clear()
+             .type('regularuser@ccrproject.dev{enter}')
+             .parents('label')
+             .should('have.class', 'q-field--error')
+                 .contains('already registered');
```

Comment:
Inconsistent indentation
```suggestion
            cy.dataCy('email_field')
                .clear()
                .type('regularuser@ccrproject.dev{enter}')
                .parents('label')
                .should('have.class', 'q-field--error')
                .contains('already registered');
```